app/components/pages/createPoll/datePicker.js

```diff
@@ -0,0 +1,45 @@
+import 'date-fns';
```

Comment:
what this line is doing here. remove this line if you are not using it

Reply:
ok